drivers/pinmux/Kconfig.npcx

```diff
@@ -0,0 +1,12 @@
+# NPCX PINMUX driver configuration options
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config PINMUX_NPCX
+	bool "Nuvoton NPCX embedd controller (EC) pinmux driver"
+	depends on SOC_FAMILY_NPCX
+	help
+	  This option enables the pin-mux driver for NPCX family 
```

Comment:
patman claims that there are a lot of trailing whitespaces, e.g. at the end of this line. Can you check?

Reply:
OK. Please notice I didn't proceed sanity check syet. I will check them later.